### PR TITLE
Queries using the guid are now executed as if indexed

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: ovs/dal/performance_test.py [hunk 1]
```diff
@@ -137,33 +147,33 @@ def test_lotsofobjects(self):
...
 
             print '\nindexed single query'
             dlist = DataList(TestDisk, {'type': DataList.where_operator.AND,
                                         'items': [('something', DataList.operator.EQUALS, repetition[0][0])]})
             start = time.time()
             assert len(dlist) == 1, 'One disk should be found'
-            seconds_passed = (time.time() - start)
-            print 'completed ({0}s) in {1} seconds (avg: {2} dps)'.format(round(time.time() - tstart, 2), round(seconds_passed, 2), round(LotsOfObjects.amount_of_machines * LotsOfObjects.amount_of_disks / seconds_passed, 2))
+            seconds_passed = time.time() - start
+            print 'completed ({0:.2f}s) in {1:.3f} seconds (avg: {2:.2f} dps)'.format(time.time() - tstart, seconds_passed, total_amount_of_disks / seconds_passed)
```
Comment (kvanhijf) [inline ovs/dal/performance_test.py:159]: 2nd formatted value has `.3f` io `.2f`

File: ovs/dal/performance_test.py [hunk 2]
```diff
@@ -175,7 +185,16 @@ def test_lotsofobjects(self):
                 assert len(repetition[i]) == len(dlist), 'Incorrect amount of found disks. Found {0} instead of {1}'.format(len(dlist), len(repetition[i]))
                 times.append(time.time() - run_start)
             seconds_passed = time.time() - start
-            print 'completed ({0}s) in {1} seconds (run avg: {2}s, avg: {3} dps)'.format(round(time.time() - tstart, 2), round(seconds_passed, 2), round(sum(times) / LotsOfObjects.repetition_scan, 2), round(LotsOfObjects.repetition_scan * LotsOfObjects.amount_of_machines * LotsOfObjects.amount_of_disks / seconds_passed, 2))
+            print 'completed ({0:.2f}s) in {1:.2f} seconds (run avg: {2:.3f}s, avg: {3:.2f} dps)'.format(time.time() - tstart, seconds_passed, sum(times) / float(LotsOfObjects.repetition_scan), LotsOfObjects.repetition_scan * total_amount_of_disks / seconds_passed)
```
Comment (kvanhijf) [inline ovs/dal/performance_test.py:188]: 3rd formatted value has `.3f` io `.2f`

**Review by kvanhijf — CHANGES_REQUESTED** [review]

File: ovs/dal/performance_test.py (hunk 1 above)
Comment (khenderick) [inline ovs/dal/performance_test.py]: That's intended. I expect smaller values there.

File: ovs/dal/performance_test.py (hunk 2 above)
Comment (khenderick) [inline ovs/dal/performance_test.py]: That's intended. I expect smaller values there.

File: ovs/dal/performance_test.py (hunk 1 above)
Comment (khenderick) [reply ovs/dal/performance_test.py:159]: That's intended. I expect smaller values there.

File: ovs/dal/performance_test.py (hunk 2 above)
Comment (khenderick) [reply ovs/dal/performance_test.py:188]: That's intended. I expect smaller values there.

**Review by kvanhijf — APPROVED** [review]

**Pull request merged**